tests/comb_sinks/README.rst

```diff
@@ -0,0 +1,6 @@
+Explicit combinational sink list
+++++++++++++++++++++++++++++++++
+
+This is an example of modeling a clock buffer which has both clock inputs and outputs. According to the "Primitive Block Timing Modeling Tutorial" section "Clock Buffers & Muxes" of the VTR manual, a clock buffer or a clock mux has to have its outputs(s) as combinational sinks of clock input(s). This contradicts with how regular sequential blocks (like flip-flops) has to be modeled. Therefore when modeling a clock buffer with V2X its clock input must have explicitly given cobinational sinks which are clock outputs.
```

Comment:
Please add links to this documentation to where in the VTR manual you are talking about.

Reply:
Ok, I've added a link to the VTR docs.